src/main/java/uk/gov/hmcts/reform/sscs/services/CaseLoaderService.java

```diff
@@ -138,19 +140,34 @@ private void processDelta(Gaps2File file) {
         log.info(logPrefixWithFile + " file transformed to {} Cases successfully", cases.size());
         int counter = 0;
         IdamTokens idamTokens = idamService.getIdamTokens();
+        processCasesFromDelta(file, cases, counter, idamTokens);
+        fileMetrics.setEndTime();
+    }
+
+    private void processCasesFromDelta(Gaps2File file, List<SscsCaseData> cases, int counter, IdamTokens idamTokens) {
         for (SscsCaseData caseData : cases) {
-            if (!caseData.getAppeal().getBenefitType().getCode().equals("ERR")) {
-                idamTokens.setServiceAuthorization(idamService.generateServiceAuthorization());
-                if (counter == numberOfProcessedCasesToRefreshTokens) {
-                    idamTokens.setIdamOauth2Token(idamService.getIdamOauth2Token());
-                    log.info(logPrefixWithFile + " renew idam token successfully");
-                    counter = 0;
-                }
+            if (caseData.getAppeal().getBenefitType().getCode().equals("ERR")) {
+                continue;
```

Comment:
if you are just continuing why don't remove the if statement?

Reply:
Here we are ignoring those cases where the benefit type code ERR